Missing dependency for beast

Boost beast also depends on endian. Compiling without it gives this error:
```
ERROR: /Users/ryanburn/proj/ot-chat-example/BUILD:1:1: C++ compilation of rule '//:ot-chat' failed (Exit 1)
In file included from src/chat_session.cpp:1:
In file included from src/chat_session.h:7:
In file included from external/boost/boost/beast/websocket.hpp:18:
In file included from external/boost/boost/beast/websocket/stream.hpp:18:
external/boost/boost/beast/websocket/detail/frame.hpp:20:10: fatal error: 'boost/endian/buffers.hpp' file not found
#include <boost/endian/buffers.hpp>
```

--- a/BUILD.boost
+++ b/BUILD.boost
@@ -197,6 +197,7 @@ boost_library(
         ":config",
         ":core",
         ":detail",
+        ":endian",
         ":smart_ptr",
         ":static_assert",
         ":system",
@@ -357,7 +358,7 @@ boost_library(
     name = "coroutine",
     srcs = BOOST_CORO_SRCS + [
         "libs/coroutine/src/detail/coroutine_context.cpp",
-        ],
+    ],
     deps = [
         ":config",
         ":context",

--- a/test/beast_test.cc
+++ b/test/beast_test.cc
@@ -3,6 +3,7 @@
 #include <boost/beast/core.hpp>
 #include <boost/beast/http.hpp>
 #include <boost/beast/version.hpp>
+#include <boost/beast/websocket.hpp>
 #include <boost/asio/connect.hpp>
 #include <boost/asio/ip/tcp.hpp>
 #include <cstdlib>

